src/blocks/content-helper/components/related-top-post-list-item.tsx

```diff
@@ -2,35 +2,35 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Button, Card, CardBody, CardHeader } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { RelatedTopPostData } from '../models/related-top-post-data';
+import ViewsIcon from '../icons/views-icon';
+import PublishedLinkIcon from '../icons/published-link-icon';
 
 interface RelatedTopPostListItemProps {
 	post: RelatedTopPostData;
 }
 
-function RelatedTopPostListItem( { post }: RelatedTopPostListItemProps ) {
+function RelatedTopPostListItem( { post }: RelatedTopPostListItemProps ): JSX.Element {
 	return (
-		<>
-			<Card size="small" elevation={ 2 } className="parsely-content-helper-card">
-				<CardHeader><b>{ post.title }</b></CardHeader>
-				<CardBody>
-					<ul>
-						<li>{ __( 'Views:', 'wp-parsely' ) } { post.views }</li>
-						<li>{ __( 'Published:', 'wp-parsely' ) } { post.date }</li>
-						<li>{ __( 'Author:', 'wp-parsely' ) } { post.author }</li>
-					</ul>
-					<p>
-						<Button href={ post.url } target="_blank" variant="primary">{ __( 'Open Post', 'wp-parsely' ) }</Button> { ' ' }
-						<Button href={ post.statsUrl } target="_blank" variant="secondary">{ __( 'Post Stats', 'wp-parsely' ) }</Button>
-					</p>
-				</CardBody>
-			</Card>
-		</>
+		<div className="parsely-top-post" data-testid="parsely-top-post">
+			<div className="parsely-top-post-title">
+				<a className="parsely-top-post-stats-link" href={ post.statsUrl } target="_blank" rel="noreferrer" title={ __( 'View in Parse.ly', 'wp-parsely' ) }>
+					{ post.title }
+				</a>
+				<a className="parsely-top-post-link" href={ post.url } target="_blank" rel="noreferrer" title={ __( 'View Published Post', 'wp-parsely' ) }>
+					<PublishedLinkIcon />
+				</a>
+			</div>
+			<ul className="parsely-top-post-info">
+				<li className="parsely-top-post-date">{ post.date }</li>
+				<li className="parsely-top-post-author">{ post.author }</li>
+				<li className="parsely-top-post-views"><ViewsIcon />{ post.views }</li>
+			</ul>
```

Comment:
This will be read out be screen readers as something like "List. 3 items. Item 1. 1st January 2022. Item 2. John Smith. Item 3: 84." That's quite verbose.

I'd rather see this marked up as just `span` elements but with some screen reader text to emulate the visual separations.

```html
<p className="parsely-top-post-info">
	<span className="parsely-top-post-date"><span className="screen-reader-text">Date </span>{ post.date }</span>
	<span className="parsely-top-post-author"><span className="screen-reader-text">Author </span>{ post.author }</span>
	<span className="parsely-top-post-views"><span className="screen-reader-text">Number of views </span><ViewsIcon />{ post.views }</span>
</p>
```

Alternatively, `dl` with `dt` (screen-reader-text) and `dd` could be used.